src/eu/janinko/games/backgammon/WrongMoveException.java

```diff
@@ -1,14 +1,21 @@
 package eu.janinko.games.backgammon;
 
+/**
+ * Exception Class for invalid Backgammon movements.
+ * 
+ * @see Exception
+ */
 public class WrongMoveException extends Exception {
-	
+
+	/**
+	 * Exception for requesting invalid moves in Backgammon. For criteria of
+	 * when this exception is used see the Backgammon rules.
+	 */
 	public WrongMoveException() {
 		super("Wrong move!");
 	}
-	/**
-	 * 
-	 */
-	private static final long serialVersionUID = 8184331891011138632L;
 
+	// Unique ID corresponding to this exception type
```

Comment:
``serialVersionUID`` is documented in https://docs.oracle.com/javase/7/docs/api/java/io/Serializable.html and as it's part of Java specification I think it's not necessary to have it commented.